src/main/java/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+public class Rectangle {
+    private int width;
+    private int height;
+
+    // "strange" constructor with no required properties, which seems to be needed for the task
+    public Rectangle() {
+    }
+
+    // auxiliary constructor with no required properties, which nice to have for short unit tests
+    public Rectangle(int width, int height) {
+        this.width = width;
+        this.height = height;
+    }
+
+    public static void main(String[] args) {
+
+        Rectangle myRect = new Rectangle();
```

Comment:
Please use rectangle constructor with properties and remove next 2 lines (or you can just remove the whole main() method, once you do it correctly already in the test method)

Reply:
Done. Rectangle constructor with appropriate properties used; redundant lines removed.